packages/core-api/src/handlers/businesses/methods.ts

```diff
@@ -16,8 +16,20 @@ const index = async request => {
 };
 
 const show = async request => {
+    let publicKey: string;
+
+    if (request.params.id.length <= 34) {
```

Comment:
I'm picky :smile: but can you do instead 
```
if (request.params.id.length === 34) {
    publicKey = request.params.id;
```
and then `else` you retrieve the public key from database service.
Would look a bit clearer to me.

Reply:
Sure, I'll turn it around, but using `66` :D